### DISCUSSION TITLE
Is it possible to deploy the Orion engine using helm?

### DISCUSSION THREAD

**Discussion by dhirschfeld** [PrefectHQ/prefect#5423] [Q&A] [answered]
Just wanting to know if this is possible, and if so, how?

**Comment by dhirschfeld** [comment]
Is it enough to set all the image tags to `master` in the [`values.yaml`](https://github.com/PrefectHQ/server/blob/master/helm/prefect-server/values.yaml)?

**Comment by zanieb** [comment, marked as answer]
Not yet — we'll likely add a Helm chart in the future and basic K8s deployments are coming soon. The Prefect Server Helm chart will not work with Orion, it has a very different set of services.

**Comment by dhirschfeld** [comment]
Where is the best place to find out when a k8s deployment is available? Is there an issue I can follow or will you post back here?

**Reply by zanieb** [reply]
We've added a kubernetes manifest / deployment in 2.0a11, here are the release notes https://github.com/PrefectHQ/prefect/blob/orion/RELEASE-NOTES.md#20a11 and here's a tutorial https://orion-docs.prefect.io/tutorials/kubernetes-flow-runner/

**Comment by dhirschfeld** [comment]
> *We've added a kubernetes manifest / deployment in 2.0a11*

💥
